tests/test_sdb.py

```diff
@@ -171,70 +171,66 @@ def test_expired_when(self):
         assert factory.is_expired(token, when=when) is True
 
 
-class TestSessionBackend(TestCase):
```

Comment:
I don't think any of the changes here are necessary?

The only thing needed here is addition of tests for the newly implemented methods.